### ISSUE TITLE
Gui not localized

### ISSUE THREAD

**Issue by LAfricain** [sahib/rmlint#488]
I use Ubuntu 20.04 and rmlint is not displayed in French. I can see the rmlint.mo file in /usr/share/locale/fr/LC_MESSAGES but the gui is not in French, only in English.

**Comment by RayOei** [comment]
It seems the translation file misses several entries for French as it does for ES. DE seems complete.

**Comment by vassilit** [comment]
The GUI is not covered by any PO file.

**Comment by RayOei** [comment]
You are correct. There doesn't seem to be any localisation for the GUI. 
I looked at the PO originally but the GUI is in python. So a different localisation approach is needed.

**Comment by vassilit** [comment]
No, the same approach (gettext) can be used with Python, but someone must nevertheless choose the strings to be translated and translate them.

**Comment by fermino** [comment]
> No, the same approach (gettext) can be used with Python, but someone must nevertheless choose the strings to be translated and translate them.

Should we try to integrate the repo with one of those translation solutions that make it easier for the community to contribute?

**Comment by vassilit** [comment]
> Should we try to integrate the repo with one of those translation solutions that make it easier for the community to contribute?

IMHO gettext is not bad to be used in git for community contributions. What do you have in mind ?

**Comment by fermino** [comment]
> > Should we try to integrate the repo with one of those translation solutions that make it easier for the community to contribute?
> 
> IMHO gettext is not bad to be used in git for community contributions. What do you have in mind ?

I was thinking of something like https://support.crowdin.com/github-integration/, without changing the current approach in the repo. It's just a way to make it easy for people to contribute.

**Comment by RayOei** [comment]
I am active in another project which tried CrowdIn. We recently decided to stop with it again. 
Contributions were non-existent to begin with and it added another layer of configuration, process and maintenance to integrate properly. We had issues when syncing non base language translation files (yaml btw) which were partly done by the maintainers before syncing with CrowdIn... There were unwanted PR's from CrowdIn. in short it gave everybody too much headache to get it working in such a way it was alleviating the work. Anyway: anecdotal evidence 😁 

I think this project is not that text heavy anyway? 
Biggest issue here is that the GUI needs updating to support multi languages as far as I can see.

I don't know whether there have been requests for more languages to support in general?

**Comment by fermino** [comment]
IMHO I don't think this is the biggest issue in the project right now, and now that I think of it the added complexity would be more annoying than helpful. De we want to try have this done for 2.10.3? Or should we leave it for 2.11?

That being said, I can take on the ES translations as it's my native language.

**Comment by RayOei** [comment]
I agree fully with it not being the biggest issue 😁 I would be fine with moving this for the 2.11 milestone or even later, depending on the other competing issues. 
As already mentioned (somewhere) by @vassilit: getting the GUI option to work without issues is most important. I guess many users aren't even able to get it to run at this moment. At least all v2.9.0 users. And after the fix I assume (almost) all macOS users are also not able to use it, based on a normal `brew` install. 
I can imagine that when it starts working for most, translation requests may drop in. They might not.
